src/internal.c

```diff
@@ -873,6 +873,7 @@ int IdentifyKey(const byte* in, word32 inSz, int isPrivate, void* heap)
     word32 idx;
     int ret;
     int dynType = isPrivate ? DYNTYPE_PRIVKEY : DYNTYPE_PUBKEY;
+    (void) dynType;
```

Comment:
Please use `WOLFSSH_UNUSED(dynType);`.

Reply:
ok, cool. the latest commit uses [WOLFSSH_UNUSED instead of (void)var](https://github.com/wolfSSL/wolfssh/pull/589/commits/56ca2d636ab31e4790ae6733da4673368d0a98cb)